src/content/posts/2020-09-29-Migrate-To-Hyperledger-Besu/2020-09-29-index-en.md

```diff
@@ -0,0 +1,68 @@
+---
+id: "2020-09-21-Migrate-to-Hyperledger-Besu"
+title: "Migrate to Hyperledger Besu: Make the switch from OpenEthereum or Multi-Geth"
+author: Kevin Lord 
+featuredImage: hyperledger_besu_etc_-750x375.jpeg
+tags: [Blog Post]
+---
+
+# Migrate to Hyperledger Besu: Make the switch from OpenEthereum or Multi-Geth
+
+![Besu-ETC-Logo](/src/content/posts/2020-09-29-Migrate-To-Hyperledger-Besu/hyperledger_besu_etc_-750x375.jpeg)
```

Comment:
Delete this line. It's a duplicate since theme already uses featuredImage in the heading section. Also, the duplicate doesn't show anyways.

Reply:
Done thanks Stev!

Reply:
Done. Thanks Stev!